apertium-eng.eng.dix

```diff
@@ -866,18 +871,21 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
   <e>       <p><l>we</l>        <r>prpers<s n="prn"/><s n="subj"/><s n="p1"/><s n="mf"/><s n="pl"/></r></p></e>
   <e r="RL"><p><l>you</l>       <r>prpers<s n="prn"/><s n="subj"/><s n="p2"/><s n="mf"/><s n="pl"/></r></p></e>
   <e>       <p><l>they</l>      <r>prpers<s n="prn"/><s n="subj"/><s n="p3"/><s n="mf"/><s n="pl"/></r></p></e>
+  <e>       <p><l>they</l>      <r>prpers<s n="prn"/><s n="subj"/><s n="p3"/><s n="mf"/><s n="sg"/></r></p></e>
   <e>       <p><l>you</l>       <r>prpers<s n="prn"/><s n="subj"/><s n="p2"/><s n="mf"/><s n="sp"/></r></p></e>
   <e r="RL"><p><l>I</l>         <r>prpers<s n="prn"/><s n="subj"/><s n="p1"/><s n="m"/><s n="sg"/></r></p></e>
   <e r="RL"><p><l>you</l>       <r>prpers<s n="prn"/><s n="subj"/><s n="p2"/><s n="m"/><s n="sg"/></r></p></e>
   <e r="RL"><p><l>we</l>        <r>prpers<s n="prn"/><s n="subj"/><s n="p1"/><s n="m"/><s n="pl"/></r></p></e>
   <e r="RL"><p><l>you</l>       <r>prpers<s n="prn"/><s n="subj"/><s n="p2"/><s n="m"/><s n="pl"/></r></p></e>
   <e r="RL"><p><l>they</l>      <r>prpers<s n="prn"/><s n="subj"/><s n="p3"/><s n="m"/><s n="pl"/></r></p></e>
+  <e r="RL"><p><l>they</l>      <r>prpers<s n="prn"/><s n="subj"/><s n="p3"/><s n="m"/><s n="sg"/></r></p></e>
   <e r="RL"><p><l>you</l>       <r>prpers<s n="prn"/><s n="subj"/><s n="p2"/><s n="m"/><s n="sp"/></r></p></e>
   <e r="RL"><p><l>I</l>         <r>prpers<s n="prn"/><s n="subj"/><s n="p1"/><s n="f"/><s n="sg"/></r></p></e>
   <e r="RL"><p><l>you</l>       <r>prpers<s n="prn"/><s n="subj"/><s n="p2"/><s n="f"/><s n="sg"/></r></p></e>
   <e r="RL"><p><l>we</l>        <r>prpers<s n="prn"/><s n="subj"/><s n="p1"/><s n="f"/><s n="pl"/></r></p></e>
   <e r="RL"><p><l>you</l>       <r>prpers<s n="prn"/><s n="subj"/><s n="p2"/><s n="f"/><s n="pl"/></r></p></e>
   <e r="RL"><p><l>they</l>      <r>prpers<s n="prn"/><s n="subj"/><s n="p3"/><s n="f"/><s n="pl"/></r></p></e>
+  <e r="RL"><p><l>they</l>      <r>prpers<s n="prn"/><s n="subj"/><s n="p3"/><s n="f"/><s n="sg"/></r></p></e>
```

Comment:
This will result in `^prpers<prn><subj><p3><f><sg>$` generating `she/they` and similarly for line 881.